### PR TITLE
[TPU] Suppress import custom_ops warning

### PR DESCRIPTION
**Pull request by WoosukKwon**
This PR suppresses the unnecessary warning when running vLLM on TPUs:
```
WARNING 08-13 04:48:22 _custom_ops.py:15] Failed to import from vllm._C with ModuleNotFoundError("No module named 'vllm._C'")
```

### CODE REVIEW

File: vllm/platforms/__init__.py
Comment (youkaichao) [inline vllm/platforms/__init__.py:1]: can you remove the change of this file to avoid merge conflict?

Comment (WoosukKwon) [reply vllm/platforms/__init__.py:1]: Let's do it after merging #7102? Without this change, the PR does not pass the mypy test.

Comment (youkaichao) [reply vllm/platforms/__init__.py:1]: #7102 merged now

Comment (WoosukKwon) [reply vllm/platforms/__init__.py:1]: Rebased.

**Review by youkaichao — APPROVED** [review]
thanks for the fix!

**Pull request merged**